.build/parent-pom-template.xml

```diff
@@ -497,6 +497,12 @@
         <version>8.40</version>
         <scope>test</scope>
       </dependency>
+			<dependency>
```

Comment:
this seems to be not aligned with other dependencies, too much tabs?

Reply:
so many tabs!